<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reflMainWidget</class>
 <widget class="QWidget" name="reflMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ISIS Reflectometry</string>
  </property>
  <layout class="QVBoxLayout" name="layoutMain">
   <item>
    <layout class="QHBoxLayout" name="layoutTemp">
     <item>
      <widget class="MantidQt::MantidWidgets::WorkspaceSelector" name="workspaceSelector">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="WorkspaceTypes">
        <stringlist>
         <string>TableWorkspace</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNew">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSaveAs">
       <property name="text">
        <string>Save As</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAddRow">
       <property name="text">
        <string>Add Row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDeleteRow">
       <property name="text">
        <string>Delete Row</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutTopRow">
     <item>
      <widget class="QLabel" name="labelInstrument">
       <property name="text">
        <string>Instrument:</string>
       </property>
       <property name="buddy">
        <cstring>comboInstrument</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboInstrument">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Sets the instrument to use.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelRuns">
       <property name="text">
        <string>Transmission run(s):</string>
       </property>
       <property name="buddy">
        <cstring>textRuns</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textRuns">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Transmission run number to be automatically entered into the 'trans' column when transferring runs.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelPolarCorrect">
       <property name="text">
        <string>Polarisation corrections:</string>
       </property>
       <property name="buddy">
        <cstring>comboPolarCorrect</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboPolarCorrect">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Sets the polarisation corrections to be done when processing.</string>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1-PNR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2-PA</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonColumns">
       <property name="text">
        <string>Choose Columns...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutMidRow">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QLabel" name="labelRB">
       <property name="text">
        <string>RB Search:</string>
       </property>
       <property name="buddy">
        <cstring>textRB</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textRB">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>The term to search the archives for</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDepth">
       <property name="text">
        <string>Search Depth (Cycles):</string>
       </property>
       <property name="buddy">
        <cstring>spinDepth</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinDepth">
       <property name="toolTip">
        <string>The depth in cycles of the serach to to on the archives. A depth of '1' seraches only the most recent cycle.</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSearch">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAuto">
       <property name="text">
        <string>AutoFill</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitterTables">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableWidget" name="tableRuns">
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Run #</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Title</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QHBoxLayout" name="layoutTableView">
       <item>
        <widget class="QPushButton" name="buttonTransfer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>=&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutButtonsTable">
         <item>
          <widget class="QTableView" name="viewTable">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ContiguousSelection</enum>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>60</number>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>20</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>20</number>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layoutButtons">
           <item>
            <widget class="QPushButton" name="buttonProcess">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Process</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonClear">
             <property name="text">
              <string>Clear all</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::WorkspaceSelector</class>
   <extends>QComboBox</extends>
   <header>MantidQtMantidWidgets/WorkspaceSelector.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboInstrument</tabstop>
  <tabstop>textRuns</tabstop>
  <tabstop>comboPolarCorrect</tabstop>
  <tabstop>buttonColumns</tabstop>
  <tabstop>textRB</tabstop>
  <tabstop>spinDepth</tabstop>
  <tabstop>buttonSearch</tabstop>
  <tabstop>buttonAuto</tabstop>
  <tabstop>tableRuns</tabstop>
  <tabstop>buttonTransfer</tabstop>
  <tabstop>viewTable</tabstop>
  <tabstop>buttonProcess</tabstop>
  <tabstop>buttonClear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
